docs/usage/guides/visual/menu.md

```diff
@@ -25,6 +25,12 @@ let menu = Menu::new()
   .add_native_item(MenuItem::Copy)
   .add_item(CustomMenuItem::new("hide", "Hide"))
   .add_submenu(submenu);
+// alternatively, using the `with_items` constructor, useful if you end up using conditional compilation
+let menu = Menu::with_items([
+  MenuItem::Copy.into(),
+  CustomMenuItem::new("hide", "Hide").into(),
+  submenu.into(),
+])
```

Comment:
I do think supplying a vector should be the default way to go. If you're making an app for multiple OS-es and one of those is macOS, then you **_should_** use conditional compilation.

Reply:
you still can do that

Reply:
Yes, but it's only provided and mentioned as an alternative. Would be nice to just have one recommended way to create a menu, instead of having one for when you want your code to look nice (?) and one for when you have cross-platform differences

Reply:
you can add entries in a vec or array conditionally, something like this

```rs

let menu = Menu::with_items([
   MenuItem::Copy.into(),
   #[cfg(target_os = "macos")]
   MenuItem::Cut.into(),
   submenu.into(),
])
```

Reply:
The problem is that we want to reduce breaking changes this close to a stable release :(

Reply:
Yes, understand that now. Feel free to merge!